packages/contracts/contracts/L1/AssetManager.sol

```diff
@@ -344,6 +359,27 @@ contract AssetManager is ISemver, IERC721Receiver, IAssetManager {
         emit Deposited(msg.sender, assets);
     }
 
+    /**
+     * @inheritdoc IAssetManager
+     */
+    function withdraw(address validator, uint128 assets) external onlyWithdrawAccount(validator) {
+        if (assets == 0) revert NotAllowedZeroInput();
+        if (VALIDATOR_MANAGER.getStatus(validator) == IValidatorManager.ValidatorStatus.NONE)
+            revert ImproperValidatorStatus();
```

Comment:
I think if the given validator is not initiated validator, it will be reverted in the condition check of `_withdraw`. So I think we don't need to check this condition, wdyt?

Reply:
That's right, I'll delete this